RealEstate/apps/core/templates/core/login.html

```diff
@@ -0,0 +1,25 @@
+{% extends "core/base.html" %}
+{% load bootstrap3 %}
+{% bootstrap_javascript jquery=1 %}
+
+{% block body %}
+  <div class="row">
+    <div class="col-sm-3"></div>
+    <div class="col-sm-6">
+      <h2 style="margin-top:50px"><strong> Login </strong> </h2>
+      <div style="margin: 10px 0" class="well" >
+      <form style="margin-bottom:20px" action="{% url 'auth_login' %}" role="form" method="post">
+    	  {% csrf_token %}
+    	  {% bootstrap_form form %}
+    	  <div class="pull-right" style="margin-top: 40px">
+    	    {% buttons submit='Log in' %}{% endbuttons %}
+    	  </div>
+    	</form>
+    	</div>
+    	{% if not state.0 and state.1 %}
```

Comment:
Was the state variable from your old login view?  If so this is no longer needed.